docs/en/modules/ws2812.md

```diff
@@ -150,6 +150,55 @@ Divide each byte of each led by the given value. Useful for a fading effect
 ```lua
 buffer:fade(2)
 ```
+## ws2812.buffer:fadeIn()
```

Comment:
How about one single method `ws2812.buffer:fade(value[, direction])`? This would keep the API lean.